modules/_labs/core/react/lib/CanvasProvider.tsx

```diff
@@ -1,12 +1,20 @@
 import * as React from 'react';
+import {ThemeProvider} from 'emotion-theming';
 import {InputProvider} from '@workday/canvas-kit-react-core';
 import {CanvasTheme} from './theming/types';
-import {ThemeProvider, defaultCanvasTheme} from './theming/theme';
+import {defaultCanvasTheme} from './theming/theme';
+import styled from '@emotion/styled';
 
 export interface CanvasProviderProps {
   theme: CanvasTheme;
 }
 
+const DirectionContainer = styled('bdo')<Pick<CanvasProviderProps, 'theme'>>(
```

Comment:
Nit: We shouldn't pull from `CanvasProviderProps`. Let's use `<{dir: ContentDirection}>`.